src/test/java/edu/hm/hafner/analysis/assertj/IssueAssert.kt

```diff
@@ -0,0 +1,147 @@
+package edu.hm.hafner.analysis.assertj
+
+import edu.hm.hafner.analysis.Issue
+import edu.hm.hafner.analysis.Priority
+import java.util.UUID
+
+import org.assertj.core.api.AbstractAssert
+
+open class IssueAssert(actual: Issue) : AbstractAssert<IssueAssert, Issue>(actual, IssueAssert::class.java) {
+
+    fun hasFileName(fileName: String): IssueAssert {
+        isNotNull
+
+        if (actual.fileName != fileName) {
+            failWithMessage("Expected issue's fileName to be <%s> but was <%s>", fileName, actual.fileName)
```

Comment:
Der Standard ist:
`private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`